File: src/dataset/python/raindrops/openAI/ChatGPT4_raindrops.py
def convert(number):
    parts = []
    if number % 3 == 0:
        parts.append("Pling")
    if number % 5 == 0:
        parts.append("Plang")
    if number % 7 == 0:
        parts.append("Plong")
    return ''.join(parts) or str(number)

File: src/dataset/typescript/raindrops_Exercism-typescript-thewanionly/openAI/ChatGPT4_raindrops.ts
export function convert(num: number): string {
  const parts: string[] = []

  if (num % 3 === 0) parts.push('Pling')
  if (num % 5 === 0) parts.push('Plang')
  if (num % 7 === 0) parts.push('Plong')

  return parts.length ? parts.join('') : String(num)
}
